test/e2e/config/conf.js

```diff
@@ -82,7 +82,7 @@ const config = {
         format: ['node_modules/cucumber-pretty'],
         require: [
             '../support/world.js',
-            '../features/step_definitions/**/*_steps.js'
+            '../features/step_definitions/**/*steps.js'
```

Comment:
should ideally have the file extension specified in the glob and kept consistent with teh rest of the app. i.e. `<step_file_name>.steps.js` then the glob would be `'../features/step_definitions/**/*.steps.js'`